### PR TITLE
Update page Kubernetes features copy

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update copy of Kubernetes features page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/kubernetes/features>
- See that the last section of the page matches the [copy doc](https://docs.google.com/document/d/1mEOBW4pfsWJqTLm9Pn9myKtaslD-QB2RvUKcNQ52bXU/edit#heading=h.b3bz7op7xuz2)

## Issue / Card

Fixes [#707](https://github.com/ubuntudesign/web-squad/issues/707)

### CODE REVIEW

File: templates/kubernetes/features.html [hunk 1]
```diff
@@ -161,8 +161,10 @@ <h2>Logging and Monitoring</h2>
 </section>
 <section class="p-strip--light">
   <div class="row">
+    <h2>Get started with Canonical&rsquo;s Distribution of Kubernetes<sup>&reg;</sup></h2>
     <p>Find out how the Canonical Distribution of Kubernetes can give you the flexibility to share operational code with a large community whilst placing your services exactly where you want them.</p>
```
Comment (anthonydillon) [inline templates/kubernetes/features.html]: Copy should be "Find out how Canonical can enable Kubernetes on demand for your devops teams -- on OpenStack, on VMware, on public clouds, and on bare metal clusters."

**Comment by frankiegranato** [comment]
@stevebeattie Ant found an error in the copydoc (my fault). Please could you update?

**Comment by steverydz** [comment]
@anthonydillon @frankiegranato Fixed

**Comment by frankiegranato** [comment]
Can we move the asterisk up and over so it is tidier please?
cdk star

**Comment by frankiegranato** [comment]
- [ ] Please make selected text into code snippet style (eg. PersistentVolumes)

- [ ] Please make the text that accompanies the grafana dashboard image into caption style (small in italics?)

**Comment by steverydz** [comment]
@frankiegranato Done

**Pull request merged**